### DISCUSSION TITLE
Is it possible for a child of a control that is a resource to reference (via a binding) a child control of a different resource?

### DISCUSSION THREAD

**Discussion by sam-wheat** [dotnet/wpf#6850] [Q&A]
In the XAML below `tb1` can be referenced by `tb2` via a binding with no problems.
However `tb4` generates a runtime error.  How can I make `tb4` bind to `tb1` in the same way as `tb2` ?

Runnable project attached.

	<Window.Resources>
        
        <StackPanel x:Key="tbResources" >
            
            <TextBox x:Name="tb1" Text="This is a resource."/>
            <TextBox x:Name="tb2" Text="{Binding Source={x:Reference tb1}, Path=Text}" /> 
            <TextBox x:Name="tb3" Text="{Binding ElementName=tb1, Path=Text}" /> 
        </StackPanel>

        <StackPanel x:Key="tbResources2">
            
            <TextBox x:Name="tb4" Text="{Binding Source={x:Reference tb1}, Path=Text}" />
        </StackPanel>
    </Window.Resources>

[ResourceTest.zip](https://github.com/dotnet/wpf/files/9193859/ResourceTest.zip)

**Comment by lindexi** [comment]
@sam-wheat We can use the resources many times. Which means that some object named tb1. And the `tb4` hard to know which object to bind to.

**Reply by sam-wheat** [reply]
@lindexi Isn't `tb1` a static resource?  If so there should only be one instance.  It seems to be working for `tb2`.  In any case, is it possible to do what I ask?
